pubspec.yaml

```diff
@@ -45,8 +52,9 @@ flutter:
   uses-material-design: true
 
   # To add assets to your application, add an assets section, like this:
-  # assets:
-  #   - images/a_dot_burr.jpeg
+  assets:
+    - assets/icons/CC-Logo.png
```

Comment:
You can just mention
``` 
assets:
    - assets/
```
to include all the resources. Mentioning each and every file could make things cumbersome.

Reply:
oh. Thanks for sharing that information.